scylla/src/transport/connection.rs

```diff
@@ -1559,8 +1558,7 @@ mod tests {
     /// 1. SELECT from an empty table.
     /// 2. Create table and insert ints 0..100.
     ///    Then use query_all with page_size set to 7 to select all 100 rows.
-    /// 3. INSERT query_all should have None in result rows.
-    /// 4. Calling query_all with a Query that doesn't have page_size set should result in an error.
+    /// 3. INSERT query_all should work and not return any rows.
     #[tokio::test]
     async fn connection_query_all_test() {
```

Comment:
Ditto with the test name: shouldn't it change its to name to `connection_query_iter_test` ? I understand that it still tests querying _all_ entities of some kind, but the function we test here is `query_iter`, so we should rather keep the name consistent with the function tested.

Reply:
Yeah, I just forgot to update the comments and the tests name. It doesn't make sense to have `query_all` anywhere in the repo at this point.

Reply:
Fixed, I also replaced the `query_all` substrings in the names of the tables created during the test.